packages/dappmanager/src/modules/installer/https.ts

```diff
@@ -5,41 +5,55 @@ import params from "../../params";
 import { InstallPackageData } from "../../types";
 import { Log } from "../../utils/logUi";
 import { HttpsPortalMapping } from "../../common";
-import { getExternalNetworkAliasFromPackage } from "../../domains";
-import { ComposeFileEditor } from "../compose/editor";
-import { dockerListNetworks, dockerCreateNetwork, dockerNetworkConnect } from "../docker";
-
+import { getExternalNetworkAlias } from "../../domains";
+import {
+  dockerListNetworks,
+  dockerCreateNetwork,
+  dockerNetworkConnect
+} from "../docker";
 
 /**
- * Persist external network on packages compose files
+ * Connect to dnpublic_network with an alias if:
+ * - is HTTPS package
+ * - any package with https portal mappings
  */
 export async function connectToPublicNetwork(
   pkg: InstallPackageData,
   externalNetworkName: string
 ): Promise<void> {
+  // if there is no https, checks aren't needed
+  if (!(await isRunningHttps())) return;
 
-  if (!(await hasRunningHTTPS())) { // if there is no https, checks aren't needed
-    return;
-  }
-
+  // create network if necessary
   const networks = await dockerListNetworks();
-  if (!networks.find(network => network.Name === externalNetworkName)) {
+  if (!networks.find(network => network.Name === externalNetworkName))
     await dockerCreateNetwork(externalNetworkName);
-  }
-
-  const containers = (await listPackageNoThrow({
-    dnpName: pkg.dnpName
-  }))?.containers || [];
 
-  for(const container of containers) {
-    if (pkg.dnpName === params.HTTPS_PORTAL_DNPNAME || await httpsPortal.hasMapping(pkg.dnpName, container.serviceName)) {
-      const alias = getExternalNetworkAliasFromPackage(
-        pkg.dnpName,
-        container.serviceName
-      );
-
-       if(!container.networks.find(n => n.name === externalNetworkName)) {
-        await dockerNetworkConnect(externalNetworkName, container.containerName, {Aliases: [alias]})
+  const containers =
+    (
+      await listPackageNoThrow({
+        dnpName: pkg.dnpName
+      })
+    )?.containers || [];
+
+  if (containers.length === 0) return;
```

Comment:
Is this necessary? As next line is pretty much the same condition.

Reply:
I guess you are right but it is a nice way of working to always double check